x-pack/plugins/security_solution/package.json

```diff
@@ -27,6 +27,7 @@
     "test:generate:serverless-dev": "NODE_TLS_REJECT_UNAUTHORIZED=0 node --no-warnings scripts/endpoint/resolver_generator --node https://elastic_serverless:changeme@127.0.0.1:9200 --kibana http://elastic_serverless:changeme@127.0.0.1:5601",
     "mappings:generate": "node scripts/mappings/mappings_generator",
     "mappings:load": "node scripts/mappings/mappings_loader",
+    "graphs:draw": "node scripts/siem_migration/draw_graphs",
```

Comment:
would it make sense to add a `siem-migrations` prefix to the script name?
`siem-migrations:graph:draw`

Reply:
Sure, I don't have any opinions on the command name itself, unless we expect more graphs later

Reply:
yes, we can change that later, for now the script is siem-migrations specific